README.md

```diff
@@ -13,29 +14,37 @@ on the node before rebooting.
 
 [Original proposal](https://docs.google.com/document/d/1DHiB2UDBYRU6QSa2e9mCNla1qBivZDqYjBVn_DvzDWc/edit#)
 
-Container Linux Update Operator is divided into two parts - `update-operator` and `update-agent`.
+Container Linux Update Operator is divided into two parts: `update-operator` and `update-agent`.
 
-`update-agent` runs on each node, waiting for a `UPDATE_STATUS_UPDATED_NEED_REBOOT` signal via dbus from `update_engine`.
+`update-agent` runs as a DaemonSet on each node, waiting for a `UPDATE_STATUS_UPDATED_NEED_REBOOT` signal via D-Bus from `update_engine`.
 It will indicate via [node annotations](./pkg/constants/constants.go) that it needs a reboot.
 
-`update-operator` will watch changes to node annotations, and reboot the nodes as needed.
+`update-operator` runs as a Deployment, watching changes to node annotations and reboots the nodes as needed.
 It coordinates the reboots of multiple nodes in the cluster, ensuring that not too many are rebooting at once.
 
 Currently, `update-operator` only reboots one node at a time.
 
 ## Requirements
 
-- Working Kubernetes >= 1.6 on CoreOS
-- `update-engine.service` should be unmasked, enabled and started in systemd
-- `locksmithd.service` should be masked and stopped in systemd
+- A Kubernetes cluster (>= 1.6) running on Container Linux
+- The `update-engine.service` systemd unit on each machine should be unmasked, enabled and started in systemd
+- The `locksmithd.service` systemd unit on each machine should be masked and stopped in systemd
+
+To unmask a service, run `systemctl unmask <name>`.
+To enable a service, run `systemctl enable <name>`.
+To start/stop a service, run `systemctl start <name>` or `systemctl stop <name>` respectively.
 
 ## Usage
 
-To start `update-operator` and `update-agent`:
+To start the `update-operator` DaemonSet, run:
 
 ```
-# Open examples/components.yaml and edit the image tag.
-kubectl create -f examples/components.yaml
+kubectl create -f examples/update-operator.yaml
```

Comment:
Oh, also, should the comment still be there?
Right now it still has "${TAG}", though I'd be in favour of updating TAG to be the latest release instead as an alternate solution.